tests/core_tests/test_context.py

```diff
@@ -1128,7 +1157,7 @@ def test_equal_yml(self):
 
     def test_equal_custom(self):
         context_1 = context.Context(**self.kws)
-        serialized = context_1.serialize(context_1)
+        serialized = context_1.serialize()
```

Comment:
This test passed because there an optionally boolean parameter for the serialize method.